spec/controllers/authors_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe AuthorsController, type: :controller do
+  describe 'POST /authors' do
+    context 'with valid params' do
+      before do
+        post :create,
+             params: {
+               first_name: 'Bill',
+               last_name: 'Gates',
+               email: 'gatess@ll.cc'
+             }
+      end
+
+      it { should render_template(:new) }
+      it { should respond_with(:created) }
+      it { should set_flash[:success].to('User data has been sent!') }
+    end
+
+    context 'with invalid params' do
+      it 'sends request with invalid params and get status 422' do
+        post :create, params: nil
+        expect(response).to have_http_status(422)
+
+        post :create, params: {
```

Comment:
I think there is no need to test every combination of wrong input here as you already covered all cases in Author model.